app/src/main/res/layout/fragment_search.xml

```diff
@@ -30,7 +30,8 @@
 
         <com.google.android.material.appbar.AppBarLayout
             android:layout_width="match_parent"
-            android:layout_height="wrap_content">
+            android:layout_height="wrap_content"
+            android:touchscreenBlocksFocus="false">
```

Comment:
Is this `touchscreenBlocksFocus="false"` necessary?

Reply:
Yes it is, if we don't use it the navigation is stuck inside the application bar